recipes-kernel/linux/files/patches-5.10/0120-arm64-dts-ti-iot2050-Add-support-for-M.2-variant.patch

```diff
@@ -107,7 +107,7 @@ index 000000000000..cb0f3a872920
 +
 +&main_gpio0 {
 +	pinctrl-names = "default";
-+	pinctrl-0 = <&main_m2_pcie_mux_control>;
++	pinctrl-0 = <&main_m2_pcie_mux_control &arduino_io_d4_to_d9_pins_default>;
```

Comment:
Just noticed: Better follow the indention pattern below, now with two entries here.

Reply:
I have considered it before. I have gone through dts of the kernel, and for most of the two entries' scenes, they wrote in one line.  if there have more entries, use multiple lines.